uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/NewFuseProjectVerifyBTNTest.java

```diff
@@ -0,0 +1,177 @@
+	
+	private static final String PROJECT_NAME = "test";
+	
+	private static final String EXISTING_CAMEL_VERSION = SupportedCamelVersions.CAMEL_2_17_0_REDHAT_630187;
+	
+	private static final String NON_EXISTING_CAMEL_VERSION = "0.0.0";
+	
+	private static final String INTERRUPT_CAMEL_VERSION = "0.0.1";
+	
+	private static final String COMMUNITY_CAMEL_VERSION = "2.20.2";
+	
+	private NewFuseIntegrationProjectWizard wizard;
+	
+	@BeforeClass
+	public static void initSetup() {
+		new WorkbenchShell().maximize();
+		ConsolePreferenceUtil.setConsoleOpenOnError(false);
+		ConsolePreferenceUtil.setConsoleOpenOnOutput(false);
+	}
+	
+	@Before
+	public void prepareTestEnvironment() {
+		wizard = new NewFuseIntegrationProjectWizard();
+		wizard.open();
+		wizard.setProjectName(PROJECT_NAME);
+		wizard.next();
+	} 
+	
+	@After
+	public void cleanSetup() {
+		LogView view = new LogView(); 
+		view.open();
+		view.deleteLog();
+		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
+		EditorHandler.getInstance().closeAll(true);
+		ProjectFactory.deleteAllProjects();
+	}
+	
+	/**
+	 * Test tries to verify existing supported Camel version
+	 */
+	@Test
+	public void testExistingCamelVersion() {	
+		wizard.setCamelVersion(EXISTING_CAMEL_VERSION);
+		wizard.verifyBTN();
+		
+		waiting();
+		finish();
+		LogChecker.assertNoFuseError();
+	}
+	
+	/**
+	 * Test tries to verify non-existing Camel version
+	 */
+	@Test
+	public void testNonExistingCamelVersion() {
+		wizard.setCamelVersion(NON_EXISTING_CAMEL_VERSION);
+		wizard.verifyBTN();
+		
+		waiting();
+		assertButtons();
+		cancel();
+		LogChecker.assertNoFuseError();
+	}
+	
+	/**
+	 * Test tries to verify community Camel version
+	 */
+	@Test
+	public void testCommunityCamelVersion() {
+		wizard.setCamelVersion(COMMUNITY_CAMEL_VERSION);
+		wizard.verifyBTN();
+		
+		waiting();
+		finish();
+		LogChecker.assertNoFuseError();
+	}
+	
+	/**
+	 * Test tries to interrupt verifying of project Camel version 
+	 */
+	@Test
+	public void testInterruptVerifying() {
+		wizard.setCamelVersion(INTERRUPT_CAMEL_VERSION);
+		wizard.verifyBTN();
+		
+		AbstractWait.sleep(TimePeriod.MEDIUM);
+		new DefaultToolItem(0).click(); // click on 'Cancel Operation'
+		waiting();
+		assertButtons();
+		cancel();
+		LogChecker.assertNoFuseError();
+	}
+	
+	/**
+	 * Verifies 'Finish' and 'Next' buttons availability  
+	 */
+	private void assertButtons() {
+		assertFalse("Button 'Next' should be disabled", wizard.isNextEnabled());
+		assertFalse("Button 'Finish' should be disabled", wizard.isFinishEnabled());
+	}
+	
+	/**
+	 * Wait until 'Finish' button is enabled and click
+	 */
+	private void finish() {
+		new WaitUntil(new ControlIsEnabled(new PushButton("Finish")), TimePeriod.MEDIUM);
+	}
+	
+	/**
+	 * Wait until 'Cancel' button is enabled and click
+	 */
+	private void cancel() {
+		new WaitUntil(new ControlIsEnabled(new PushButton("Cancel")), TimePeriod.MEDIUM);
```

Comment:
Please, use referencedComposite in PushButton. Use can use CancelButton class instead of PushButton